README.md

```diff
@@ -46,6 +46,7 @@ This generator can also be further configured with the following command line fl
     -H, --hogan          add hogan.js engine support
     -v, --view <engine>  add view <engine> support (dust|ejs|hbs|hjs|jade|pug|twig|vash) (defaults to jade)
         --no-view        use static html instead of view engine
+    -a, --api            use web api template
```

Comment:
The docs don't seem to mention what users should expect under the various combinations of command line options here, like I assume that passing in `--api --pug` make the pug do nothing? Same for CSS. Just needs clarification on how this interacts with the other options somewhere.